docs/dashboard/orders.md

```diff
@@ -14,6 +14,8 @@ There are essentially two order types in Saleor:
 
 - Draft orders - These are orders created by administrators, which have not yet been released to the system. For example, store operators can create orders during discussions with customers, then save them as drafts until approval is gained from management to send replacement goods. Draft orders can be edited in the same way that an order is created.
 
+- Unconfirmed orders - Orders placed by customers when order confirmation feature is turned on. The payment on such orders is being authorized when order is placed but captured later - when staff [confirms the order](#how-to-confirm-an-order).
```

Comment:
"when order confirmation feature is turned on" - this part could link to the section where it is described how to turn on/off order confirmation.